Content/GUI/ExpBar.cs

```diff
@@ -16,14 +16,6 @@ public override int InsertionIndex(List<GameInterfaceLayer> layers)
 	{
 		return layers.FindIndex(layer => layer.Name.Equals("Vanilla: Mouse Text"));
 	}
-	
-	public Rectangle GetRectangle()
-	{
-		Texture2D bar = ModContent.Request<Texture2D>($"{PathOfTerraria.ModName}/Assets/GUI/BarEmpty").Value;
-		var pos = new Vector2(Main.screenWidth / 2, 10);
-
-		return new Rectangle((int)(pos.X - bar.Width / 2f), (int)pos.Y, bar.Width, bar.Height);
-	}

```

Comment:
After removing this the XP bar seems to work just fine and didn't notice a single issue with all sorts of resizing so may have just been extra